src/elements/content-sidebar/withSidebarAnnotations.js

```diff
@@ -83,6 +83,12 @@ export default function withSidebarAnnotations(
             if (prevFileVersionId !== fileVersionId) {
                 this.updateActiveVersion();
             }
+
+            if (prevFileId !== fileId) {
```

Comment:
Good point, I don't think it would because on a file change the new `fileVersionId` should be `undefined` which should lead to no version being found -- but I can change the expression to `fileVersionid && prevFileVersionId !== fileVersionId` to avoid even calling into `this.updateActiveVersion`